app/main/plugins/core/basic-apps/linux.js

```diff
@@ -0,0 +1,106 @@
+import { remote } from 'electron'
+import path from 'path'
+import fs from 'fs'
+import uniq from 'lodash/uniq'
+import flatten from 'lodash/flatten'
+import { shellCommand } from 'cerebro-tools'
+
+let appDirs = [
+  path.join(remote.app.getPath('home'), '.local', 'share'),
+  path.join('/usr', 'share'),
+  path.join('/usr', 'share', 'ubuntu'),
+  path.join('/usr', 'share', 'gnome'),
+  path.join('/usr', 'local', 'share'),
+  path.join('/var', 'lib', 'snapd', 'desktop')
+]
+
+if (!!process.env.XDG_DATA_DIRS) {
+  appDirs = [
+    ...appDirs,
+    ...process.env.XDG_DATA_DIRS.split(':')
+  ]
+}
+
+// Icon resolutions in priority of checking
+const iconResolutions = [
+  '128x128',
+  '96x96',
+  '64x64',
+  '256x256',
+  '512x512'
```

Comment:
I know that you have `max-width: 200px` anyway, but to me 512px scaled down to 200px still looks nicer than 64px.

To make the point, an example of 64x64 vs 256x256:

![cerebro_288](https://cloud.githubusercontent.com/assets/1177900/22529577/3afc9bb0-e8d8-11e6-85c2-cc11b212f880.png)

![cerebro_289](https://cloud.githubusercontent.com/assets/1177900/22529578/3faa4c2a-e8d8-11e6-88ed-3dbff6cc9dec.png)